pjsip/src/pjsip-ua/sip_100rel.c

```diff
@@ -269,6 +269,12 @@ PJ_DEF(pj_status_t) pjsip_100rel_create_prack( pjsip_inv_session *inv,
     }
     rseq = (pj_uint32_t) pj_strtoul(&rseq_hdr->hvalue);
 
+    if (rseq < 1) {
+        PJ_LOG(4, (dd->inv->dlg->obj_name,
+                   "Ignoring 100rel response RSeq header value less than 1"));
+        return PJSIP_EINVALIDMSG;
+    }
+
```

Comment:
Actually as receiver, we better be lenient, especially if it is not so urgent/risky, perhaps printing warning is sufficient?

Reply:
If the message received with RSeq 0, currently it will be ignored in https://github.com/pjsip/pjproject/blob/coverity02/pjsip/src/pjsip-ua/sip_100rel.c#L303

Reply:
so should we return PJ_EIGNORED here, so it's the same as the old behavior?

Reply:
> If the message received with RSeq 0, currently it will be ignored in https://github.com/pjsip/pjproject/blob/coverity02/pjsip/src/pjsip-ua/sip_100rel.c#L303

The pointed code does not seem to check for `rseq==0`, it checks for retransmission (where the original rseq may still be 0)? So IMO, perhaps better check the behavior when `rseq ==0` (e.g: using SIPp) to at least avoid behavior change.

Reply:
The rseq is unsigned, so the condition will always be true when `rseq ==0` regardless the original rseq. If we want to avoid behavior change we should return PJ_EIGNORED (as @sauwming suggested) or just print the warning message.